### PR TITLE
Responsive layout

### PR DESCRIPTION
**Pull request by FritzHoing**
The language-selektor has been moved into the Toolbox. 

Layout for a Smartphone:
Auswahl_001
Layout for a TouchPad:
Auswahl_002

### CODE REVIEW

File: src/components/Header/index.less [hunk 1]
```diff
@@ -50,22 +44,82 @@
...
-      }
     }
+
     &.right-items {
       flex: 1;
       justify-content: flex-end;
     }
+  }
+
+  @media (max-width:599px) {
```
Comment (LukasLohoff) [inline src/components/Header/index.less:55]: is this for smartphones or tablets or both? I think a short comment explaining that would be useful here

**Review by Unraveler — APPROVED** [review]
Looks good to me

File: src/components/Header/index.less [hunk 2]
```diff
@@ -50,22 +44,82 @@
...
+        width: fit-content;
+      }
+    }
+
+    .item-container.right-items {
+      width: 100px;
+      max-width: 100px;
+    }
+  }
+}
```
Comment (LukasLohoff) [inline src/components/Header/index.less:125]: missing newline at end of file. Do you have the `editorconfig` plugin installed?

Comment (LukasLohoff) [reply src/components/Header/index.less:125]: that should add these automatically, and also configure other editor settings for you (whitespace indentation, etc.)

File: src/components/LanguageSelector/index.less [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.languageSelect {
```
Comment (LukasLohoff) [inline src/components/LanguageSelector/index.less:1]:
```suggestion
.language-select {
```

Comment (LukasLohoff) [reply src/components/LanguageSelector/index.less:1]: we usually format css classes like this. if you accept this, you'll also have to adjust the `LanguageSelect` component though

**Review by LukasLohoff — CHANGES_REQUESTED** [review]
Nice! I suggested some small changes, maybe you can have a look at the comments below.

**Review by LukasLohoff — APPROVED** [review]

File: src/components/LanguageSelector/index.less [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.language-select {
+  display: flex;
+  justify-content: center;
+
+  .ant-select {
+    width: 100%;
+    margin: 5%;
```
Comment (dnlkoch) [inline src/components/LanguageSelector/index.less:7]: Is `5%` intended here?

File: src/components/Header/index.tsx [hunk 5]
```diff
@@ -45,7 +45,6 @@ export const Header: React.FC<HeaderProps> = ({
       <div
         className="item-container right-items"
       >
-        <LanguageSelect />
```
Comment (dnlkoch) [inline src/components/Header/index.tsx:48]: Can we remove the import as well?

File: src/components/LanguageSelector/index.less [hunk 6]
```diff
@@ -0,0 +1,9 @@
+.language-select {
+  display: flex;
+  justify-content: center;
```
Comment (dnlkoch) [inline src/components/LanguageSelector/index.less:3]: Is this really needed?

File: src/components/ToolMenu/index.tsx [hunk 7]
```diff
@@ -304,6 +306,22 @@ export const ToolMenu: React.FC<ToolMenuProps> = ({
...
+      label: <Menu.Divider />
+    },
+    {
+      key: 'LanguageSelect',
+      onTitleClick: onSubmenuTitleClick,
+      icon: <FontAwesomeIcon icon={faLanguage} />,
+      label: t('ToolMenu.languageSelect'),
+      children: [
+        {
+          key: 'LanguageSelect',
```
Comment (dnlkoch) [inline src/components/ToolMenu/index.tsx:320]: The key is used in the surrounding block as well.

File: src/components/LanguageSelector/index.less (hunk 4 above)
Comment (FritzHoing) [reply src/components/LanguageSelector/index.less:7]: Yes, this is the margin of the dropdown-menu:
Auswahl_003

Comment (dnlkoch) [reply src/components/LanguageSelector/index.less:7]: Yes, but why it's relative?

File: src/components/LanguageSelector/index.less (hunk 6 above)
Comment (FritzHoing) [reply src/components/LanguageSelector/index.less:3]: Indeed, justify-content can be removed. But display: flex is needed. If it is missing, the drop down-menu is not completely within the toolbox.

File: src/components/LanguageSelector/index.less (hunk 4 above)
Comment (FritzHoing) [reply src/components/LanguageSelector/index.less]: You are right! An absolute margin value makes more sense.

**Comment by weskamm** [comment]
:heart:

**Review by dnlkoch — APPROVED** [review]

**Pull request merged**